mlflow/server/handlers.py

```diff
@@ -2046,13 +2064,30 @@ def _create_model_version():
         description=request_message.description,
         model_id=request_message.model_id,
     )
-    if not _is_prompt_request(request_message) and request_message.model_id:
+    if not is_prompt and request_message.model_id:
         tracking_store = _get_tracking_store()
         tracking_store.set_model_versions_tags(
             name=request_message.name,
             version=model_version.version,
             model_id=request_message.model_id,
         )
+
+    if not is_prompt:
+        try:
+            dispatch_webhook(
+                event=WebhookEvent.MODEL_VERSION_CREATED,
+                payload={
+                    "name": request_message.name,
```

Comment:
Can't we use typed dicts defined in `mlflow/webhooks/__init__.py` here to make sure the same schema is used in the business logic?